modules/k6/examples_test.go

```diff
@@ -0,0 +1,73 @@
+package k6_test
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/k6"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func ExampleRunContainer() {
+	// runK6Container {
+	ctx := context.Background()
+
+	gcr := testcontainers.GenericContainerRequest{
```

Comment:
Could you include some comments to the method signature including why we need the httpbin application? I guess it's for https://github.com/testcontainers/testcontainers-go/pull/1721/files#diff-d82d09415608143910841e29e49f3b7e85d98c3d6e7def8a44468a5ffb1732abR5, but want to be more explicit in the comments for this testable example 🙏